config/default/common/config/metadata/stories/hurricane_maria_september_2017/step003.md

```diff
@@ -1 +1 @@
-<p>Here we've added the Rain Rate layer to show the rain rates associated with the storm. You can change the opacity of this layer so that you can still see the imagery underneath. You can also change the color palette by clicking on the "Layer Options" icon.</p>
+<p>Here we've added the Precipitation Rate layer to show the rain rates associated with the storm. You can change the opacity of this layer so that you can still see the imagery underneath. You can also change the color palette by clicking on the "Layer Options" icon.</p>
```

Comment:
```suggestion
Here we've added the Precipitation Rate layer to show the rain rates associated with the storm. You can change the opacity of this layer so that you can still see the imagery underneath. You can also change the color palette by clicking on the "Layer Options" icon.
```